index.bs

```diff
@@ -671,12 +671,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     An [=authorization gesture=] is a [=ceremony=] component often employed to indicate [=user consent=].
 
 : <dfn>User Handle</dfn>
-:: The user handle is specified by a [=[RP]=] and is a unique identifier for a user account with that [=[RP]=]. A user handle is
-    an opaque [=byte sequence=] with a maximum size of 64 bytes.
+:: The user handle is specified by a [=[RP]=] and used to map a specific [=public key credential=]
+    to a specific user account with the [=[RP]=]. Authenticators in turn [=credentials map|map=] [=RP ID=]s and user handle pairs
+    to [=public key credential sources=].
 
-    The user handle is not meant to be displayed to the user, but is used by the [=[RP]=] to control the number of credentials - an
-    authenticator will never contain more than one credential for a given [=[RP]=] under the same user handle.
+    Note: [=[RPS]=] may map multiple [=public key credentials=] to a user account by mapping multiple user handles to the account.
```

Comment:
Perhaps this could be better expressed as something like this?

>Note: [=[RPS]=] MAY map multiple [=public key credentials=] to one user account by supplying the same [=user handle=] as the \<code>{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/id}}\</code> argument in multiple calls to {{CredentialsContainer/get()}}.

Reply:
I tried putting that in to see that it compiles, so I suppose I might as well push it as well: #1053

Reply:
Hm, maybe I'm missing something, but given how the authnr's [credentials map](https://w3c.github.io/webauthn/#authenticator-credentials-map) is defined -- a (rpId, \[[userHandle](https://w3c.github.io/webauthn/#public-key-credential-source-userhandle)\]) pair yields a [credential source](https://w3c.github.io/webauthn/#public-key-credential-source) -- does that work?

Reply:
Ah, good point. So that would be

>...MAY map multiple [=public key credential sources=] to one user account...

then?